contrib/comfort_models.py

```diff
@@ -171,67 +215,86 @@ def comfPMV(ta, tr, vel, rh, met, clo, wme):
 
     ts = 0.303 * math.exp(-0.036 * m) + 0.028
     pmv = ts * (mw - hl1 - hl2 - hl3 - hl4 - hl5 - hl6)
-    ppd = 100.0 - 95.0 * math.exp(-0.03353 * pow(pmv, 4.0)
-        - 0.2179 * pow(pmv, 2.0))
+    ppd = 100.0 - 95.0 * math.exp(-0.03353 * pow(pmv, 4.0) - 0.2179 * pow(pmv, 2.0))
 
-    r = []
-    r.append(pmv)
-    r.append(ppd)
+    # todo I would recoommend to return json instead
```

Comment:
You can remove the todo.. I don't think we want JSON here, maybe you mean a python dict?

Reply:
Yes that is what I meant. Removing the TODOs and moving them to an issue I believe would make it more difficult to interpret them. Leaving the TODOs in line with the code, helps me to find where (e.g. in which part of the code) I need to address the issue.

Reply:
I think it's the other way around.. you'd have to search the code for "TODO" to find it. You could put a line number in the issue if it was in a place that's difficult to remember, but the code is not the best place to keep track of work you need to do later, unless it's related to a specific quirk in the code. If you really want to keep it here, at least make it accurate and a bit more descriptive. Remember JSON would be a string, and I don't think that's what you mean here.. I think you mean "a dict with keys 'pmv' and 'ppd'"